src/Bundle/CoreBundle/Resources/views/Form/uikit_layout.html.twig

```diff
@@ -55,6 +55,18 @@
     {{- parent() -}}
 {%- endblock textarea_widget -%}
 
+{%- block link_widget -%}
```

Comment:
You don't need to iterate over child widgets, symfony form does this for you. But you don't need to change anything here, since I will do the frontend part. Just an info :)